### PR TITLE
Software part 1 #54 #45

### PR DESCRIPTION
**Pull request by cmeesters**
Motivation:

- need to fall back to Mamba - no Micromamba support for Snakemake, yet
- the rest of the world does not suffer from the peculiarities of "my" cluster

### CODE REVIEW

**Comment by cmeesters** [comment]
Do not merge, yet. This is a draft.

**Comment by cmeesters** [comment]
Alas, this PR includes additional parts:
- some slides about Snakemake
- corrections / amendments to render script
- minor additions elsewhere

File: slides/creators/Software_Provisioning_with_Snakemake.tex [hunk 1]
```diff
@@ -171,7 +171,7 @@ \subsection{Environment Modules}
   \frametitle{Using Environment Modules II}
   When \Snakemake{} is executed with 
   \begin{lstlisting}[language=Bash, style=Shell]
-snakemake --use-envmodules
+snakemake --sdm envmodules
```
Comment (mptrsen) [inline slides/creators/Software_Provisioning_with_Snakemake.tex]: Should be `env-modules` according to [docs](https://snakemake.readthedocs.io/en/v8.2.3/executing/cli.html#software-deployment)

**Review by mptrsen — APPROVED** [review]
Great :+1:

**Pull request merged**